lldb/source/Commands/Options.td

```diff
@@ -841,6 +841,12 @@ let Command = "scripting run" in {
     " language. If none is specific the default scripting language is used.">;
 }
 
+let Command = "scripting template list" in {
+  def scripting_template_list_language : Option<"language", "l">,
+    EnumArg<"ScriptLang">, Desc<"Specify the scripting "
+    " language. If none is specific the default scripting language is used.">;
```

Comment:
specific -> specified

Reply:
I don't think that's what is currently implemented, however, since you reset the language to None, and then error out if it is...